photon-client/src/main/scala/com/linkedin/photon/ml/util/IOUtils.scala

```diff
@@ -342,7 +259,7 @@ protected[ml] object IOUtils {
    * which can itself be either Success or Failure, wrapping an instance of the type returned by the lambda.
    * Failure.map(lambda) ignores lambda, and returns itself, but changing the contained type to the type
    * returned by the lambda (see scala.util.Try). Failure thus contains an exception, if one is thrown.
-   * On the last line, flatMap is used to avoid returning Try[Try[Unit]].
+   * On the last line, flatMap is used to avoid returning Try[Try[Unit] ].
```

Comment:
It is actually correct Scala syntax, is it not? If I leave `]]` the IDE flags it in red because it thinks it's one of those (unreadable) comment reference thing. I removed the line explaining the semantics of `flatMap` altogether. We can assume somebody reading this knows what `flatMap` does.